packages/client/src/runtime/core/engines/common/errors/engine-not-found/toolingHasTamperedWithEngineCopy.ts

```diff
@@ -0,0 +1,17 @@
+import { weWouldAppreciateIfYouCouldShareInfo } from '../weWouldAppreciateIfYouCouldShareInfo'
+import { EngineNotFoundErrorInput } from './EngineNotFoundErrorInput'
+import { queryEngineCannotBeFound } from './queryEngineCannotBeFound'
+import { theseLocationsHaveBeenSearched } from './theseLocationsHaveBeenSearched'
+
+export function toolingHasTamperedWithEngineCopy(input: EngineNotFoundErrorInput) {
+  const { queryEngineName } = input
+
+  return `${queryEngineCannotBeFound(input)}
+
+This is likely caused by tooling that has not copied "${queryEngineName}" to the deployment.
```

Comment:
`to the deployment` what? Folder? Target? I'd add a noun after `deployment` to make the sentence clearer to users :)

Reply:
I'll add "to the deployment folder", that will be clearer.